hack/ci/operator-ensure-manifests.sh

```diff
@@ -13,30 +13,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Script responsible for ensuring and correcting manifests as needed.
+set -e
 
-source ./hack/env.sh
+source ./hack/ci/ensure-image.sh
```

Comment:
why do you need to `source` there ?
any variable needed ?

Reply:
It is taking IMAGE and KIND_IMAGE vars from ensure-image, also since ensure-image.sh calls env.sh we can have its function in this script like `getLatestOlmReleaseVersion`